list_and_dictionary.py

```diff
@@ -0,0 +1,105 @@
+lloyd = {
+  "name": "Lloyd",
+  "homework": [90.0,97.0,75.0,92.0],
+  "quizzes": [88.0,40.0,94.0],
+  "tests": [75.0,90.0]
+}
+alice = {
+  "name": "Alice",
+  "homework": [100.0, 92.0, 98.0, 100.0],
+  "quizzes": [82.0, 83.0, 91.0],
+  "tests": [89.0, 97.0]
+}
+tyler = {
+  "name": "Tyler",
+  "homework": [0.0, 87.0, 75.0, 22.0],
+  "quizzes": [0.0, 75.0, 78.0],
+  "tests": [100.0, 100.0]
+}
+
+## print the student's name
+
+student = [lloyd.get("name",),alice.get("name"),tyler.get("name")]
```

Comment:
extra `,` after `lloyd.get("name"`

I notice that you really like using `.get`. Just FYI `lloyd["name"]` would do the same thing if you haven't tried